hubblestack/files/hubblestack_nova/oval_scanner.py

```diff
@@ -166,21 +187,50 @@ def build_impact_report(impact, report={}):
     return report
 
 
-def get_impact(local_ver, name, ver, title, cve, advisory, severity):
+def get_impact(local_ver, distro_name, **kargs):
     """Compare local package ver to vulnerability ver in rpm distros"""
     logging.debug('get_rpm_impact')
     impact = {}
-    if __mods__['pkg.version_cmp'](ver, local_ver) > 0:
-        impact[title] = {
-            'updated_pkg': {'name': name, 'version': ver},
-            'installed': {'name': name, 'version': local_ver},
-            'severity': severity,
-            'advisory': advisory,
-            'cve': cve
-        }
+    if __mods__['pkg.version_cmp'](kargs['ver'], local_ver) > 0:
+        impact = create_impact(impact, local_ver, **kargs)
+    return impact
+
+
+def create_impact(impact, local_ver, **kargs):
+    """create impact based on local version"""
+    logging.debug('create_impact')
```

Comment:
Again, can we add the variable **impact** in the log statement. This would actually help in the debugging later. Or do you think that will be unnecessary noise?

Reply:
I've added this as a debug log.